src/App.svelte

```diff
@@ -1,46 +1,96 @@
 <svelte:head>
-	<style src="./bulma.scss"></style>
+    <style src="./bulma.scss"></style>
 </svelte:head>
 
 <script lang="ts">
-	import SettingUi from './components/settings/SettingsUi.svelte'
-	import DataDisplay from './components/data-display/MainDataDisplay.svelte'
-	import TimeOnTargetChart from './components/time-on-target-chart/TimeOnTargetChart.svelte'
+    import SettingUi from './components/inputs/SettingsUi.svelte'
+    import Mttot from './components/Mttot.svelte';
+    import { NavbarTabs } from './typescript/Navbar';
+
+    let selectedTab = NavbarTabs.MTTOT;
+    let newTab = NavbarTabs.MTTOT;
+
+    const selectTab = (): void => {
+        if (newTab === selectedTab) {
+            return;
+        }
+        document.getElementById(`${selectedTab}Nav`).classList.remove('is-active');
+        document.getElementById(`${newTab}Nav`).classList.add('is-active');
+        selectedTab = newTab;
+    };
+
+    $: newTab, selectTab();
+
 </script>
 
-<main class="is-flex is-justify-content-center">
-	<div class="grid-container vertically-center">
-		<div class="component-container">
-			<SettingUi/>
-		</div>
+<div class="main-page-container">
+    <div class="tabs is-boxed is-fullwidth mb-0 tab-container">  <!-- width of tab-buttons needs to be changed depending on how many there are -->
+        <ul class="p-0 m-0 ml-3 mr-3">
+            <li id="mttotNav" class="tab is-active" on:click={() => newTab = NavbarTabs.MTTOT}>
+                <!-- svelte-ignore a11y-missing-attribute -->
+                <a><span>Mttot Calculator</span></a>
+            </li>
+            <li id="gaussNav" class="tab" on:click={() => newTab = NavbarTabs.GAUSS}>
+                <!-- svelte-ignore a11y-missing-attribute -->
+                <a><span>Gauss Calculator</span></a>
+            </li>
+        </ul>
+    </div>
 
-		<div class="component-container inner-container">
-			<DataDisplay/>
-		</div>
+    <div class="is-flex content-container">
+        <div class="setting-container">
+            <SettingUi/>
+        </div>
 
-		<div class="component-container">
-			<TimeOnTargetChart/>
-		</div>
-	</div>
-</main>
+        {#if selectedTab === 'mttot'}
```

Comment:
There are some improvements to be done to the navigation tabs in my opinion. I would say it can be dealt with separately, so it doesn't need to get done in this PR